Makefile

```diff
@@ -1,5 +1,3 @@
-PROJECT_NAME := policy
-SUB_PROJECTS := sdk/nodejs/policy sdk/python
```

Comment:
I think we need to keep these two lines for `build/common.mk`.

Reply:
Whoops, got overzealous in cleaning there.